packages/markdown/components/Image.jsx

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable react/jsx-props-no-spreading */
+
+const React = require('react');
+const PropTypes = require('prop-types');
+
+const Image = props => {
+  const [title, align, width = 'auto', height = 'auto'] = props.title
+    ? props.title.split(', ')
+    : [];
+  const extras = { align, width, height };
+  if (props.caption)
+    return (
+      <figure {...extras} style={{ width: extras.width }}>
+        <img {...props} alt={props.alt} title={title} />
+        <figcaption>{props.caption}</figcaption>
+      </figure>
+    );
+  return <img {...props} alt={props.alt} title={title} {...extras} />;
+};
+
+Image.propTypes = {
+  align: PropTypes.string,
+  alt: PropTypes.string,
+  caption: PropTypes.oneOfType([PropTypes.string, PropTypes.bool]),
+  height: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+  src: PropTypes.string,
```

Comment:
I assume you mean the `caption` prop? No, it's just a placeholder for extra data from magic image blocks, since pure image captions aren't a feature of standard markdown syntax!

Reply:
No, this was in reference to `src` not currently being declared as a required prop.

Reply:
It's required now.